tests/data/group_system/group_accounts/group_accounts-restrictions/group_password_expiration/rule_accounts_password_minlen_login_defs/password_minlen_commented.fail.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# platform = multi_platform_fedora,Red Hat Enterprise Linux 8
+
+sed -i "s/.*PASS_MIN_LEN.*/#PASS_MIN_LEN 12/" /etc/login.defs
```

Comment:
Just a minor thing - change `12` to `15` here (or more). Because it can happen that the check is not testing comments, but only number after `PASS_MIN_LEN` and in this test scenario it will give ok result (expected fail), but because of the number under limit and not because of `#`.

Reply:
well observed, but we can keep it like that because of value `12` in new OSPP profile.